plugins/techdocs/src/EntityPageDocs.tsx

```diff
@@ -17,13 +17,21 @@
 import React from 'react';
 import { Entity } from '@backstage/catalog-model';
 import { Reader } from './reader';
+import { configApiRef, useApi } from '@backstage/core-plugin-api';
 
 export const EntityPageDocs = ({ entity }: { entity: Entity }) => {
+  // Lower-case entity triplets by default, but allow override.
+  const toLowerMaybe = useApi(configApiRef).getOptionalBoolean(
+    'techdocs.legacyUseCaseSensitiveTripletPaths',
+  )
+    ? (str: string) => str
+    : (str: string) => str.toLocaleLowerCase('en-US');
+
   return (
     <Reader
       withSearch={false}
       entityRef={{
-        kind: entity.kind,
+        kind: toLowerMaybe(entity.kind),
         namespace: entity.metadata.namespace ?? 'default',
         name: entity.metadata.name,
```

Comment:
I believe we want to run namespace and name through it as well, similar to how its used here https://github.com/backstage/backstage/blob/7439368ce532e98bea8994b2be1899c9af9626ca/plugins/techdocs/src/home/components/DocsTable.tsx#L70-L72

Reply:
if you update this you can also take the opportunity to update the name of the changeset to be prefixed with techdocs-*, that way we can focus on the techdocs-core team to review it!